README.md

```diff
@@ -0,0 +1,53 @@
+# batwatch
+
+_batwatch_ monitors a system's batteries and UPS units for state changes and sends
+notifications via _gpgmailer_.
+
+Also depends on our _confighelper_ library which can be found at
+https://github.com/park-bench/confighelper
+
+batwatch is licensed under the GNU GPLv3. All source code commits prior to the public
+release are also retroactively licensed under the GNU GPLv3.
```

Comment:
There are no commits prior to the public release. You can remove the second part of this paragraph.